label_studio/webhooks/models.py

```diff
@@ -0,0 +1,180 @@
+import requests
+from core.validators import JSONSchemaValidator
+from django.db import models
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+from django.conf import settings
+import logging
+
+from projects.models import Project
+from tasks.models import Task, Annotation
+from .serializers_for_hooks import (
+    OnlyIDWebhookSerializer,
+    ProjectWebhookSerializer,
+    TaskWebhookSerializer,
+    AnnotationWebhookSerializer,
+)
+
+HEADERS_SCHEMA = {
+    "type": "object",
+    "patternProperties": {
+        "^[a-zA-Z0-9-_]+$": {"type": "string"},
+    },
+    "maxProperties": 10,
+    "additionalProperties": False,
+}
+
+
+class Webhook(models.Model):
+    """Model of webhooks.
+
+    If webhook has not null project field -- it's project webhook
+    """
+
+    organization = models.ForeignKey('organizations.Organization', on_delete=models.CASCADE, related_name='webhooks')
+
+    project = models.ForeignKey(
+        'projects.Project', null=True, on_delete=models.CASCADE, related_name='webhooks', default=None
+    )
+
+    url = models.URLField(_('URL of webhook'), max_length=2048)
```

Comment:
`help_text=` is needed here, in every non-reference fields

Reply:
did it